DataFormats/PatCandidates/interface/PackedCandidate.h

```diff
@@ -664,13 +652,26 @@ namespace pat {
     reco::VertexRef::key_type pvRefKey_;
 
     /// IP covariance	
-    CMS_THREAD_GUARD(vertex_) mutable float dxydxy_, dzdz_, dxydz_,dlambdadz_,dphidxy_,dptdpt_,detadeta_,dphidphi_;
+    CMS_THREAD_GUARD(vertex_) mutable reco::TrackBase::CovarianceMatrix m_;
```

Comment:
This is not being guarded by `vertex_`. Instead, it appears that the intent is to guard using `track_`. However, that protection isn't correct since `m_` isn't just getting the same value assigned but instead is first reset and then gets its value.